core-addon/DataCollection.js

```diff
@@ -155,6 +155,11 @@ module.exports = class DataCollection {
       schemaNamespace: namespace,
     };
 
+    if (!__ENABLE_DATA_SUBMISSION__) {
+      console.log(`DataCollection.sendPing - data submission disabled, ping ${payloadType} not submitted`);
```

Comment:
nit but I'd consider using `console.warn` here since this is moderate severity, we want folks to notice this so it's not accidentally shipped this way etc.